### PR TITLE
[sht] Added bash autocomplete

### PR DESCRIPTION
**Pull request by Jacob-C-Smith**
I added autocomplete, updated the README with instructions, and removed the corresponding TODO

### CODE REVIEW

File: sht_complete.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            COMPREPLY=($(compgen -W "-f --force" -- "${COMP_WORDS[2]}"))
+            ;;
+        *)
+            COMPREPLY=($(compgen -W "init status store check check-tree wipe normalize-files hash" -- "$command"))
+            ;;
+    esac
+}
+
+# Register the autocomplete function with the command
+complete -F _mycontext_complete sht
```
Comment (Jacob-C-Smith) [inline sht_complete.sh:38]: complete command lets you do bash autocomplete. -F is for function. Command itself is last parameter.

File: sht_complete.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            COMPREPLY=($(compgen -W "$(ls)" -- "${COMP_WORDS[2]}"))
+            ;;
+        wipe)
+            COMPREPLY=($(compgen -W "$(ls)" -- "${COMP_WORDS[2]}"))
+            ;;
+        normalize-files)
+            COMPREPLY=($(compgen -W "$(ls)" -- "${COMP_WORDS[2]}"))
+            ;;
+        hash)
+            COMPREPLY=($(compgen -W "-f --force" -- "${COMP_WORDS[2]}"))
```
Comment (Jacob-C-Smith) [inline sht_complete.sh:29]: Not sure if --force has been implemented yet

File: sht_complete.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Define the command completion function
+_mycontext_complete() {
+    local command="${COMP_WORDS[1]}"
+    
+    case "$command" in
+        init)
+            COMPREPLY=($(compgen -W "" -- "${COMP_WORDS[2]}"))
+            ;;
+        status)
+            COMPREPLY=($(compgen -W "--recursive" -- "${COMP_WORDS[2]}"))
```
Comment (Jacob-C-Smith) [inline sht_complete.sh:10]: Not sure if --recursive has been implemented yet, but better to be proactive than reactive.

**Comment by ben256dev** [comment]
Excellent work Jacob! I really like that you even updated the README.

**Pull request merged**